src/main/java/de/tum/in/www1/artemis/domain/Course.java

```diff
@@ -109,6 +110,11 @@ public class Course extends DomainObject {
     @JoinColumn(name = "online_course_configuration_id")
     private OnlineCourseConfiguration onlineCourseConfiguration;
 
+    @Enumerated(EnumType.ORDINAL)
```

Comment:
~~Why is an ordinal used here instead of the enum name like in most other cases?~~

~~As the number of courses in an instance is comparatively small, I’d not expect the additional storage requirement to make much difference here.~~

Re-read the PR description.